harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/geoPREST/Harvester.java

```diff
@@ -262,6 +265,33 @@ private RecordInfo getRecordInfo(Element record) {
 
     }
 
+    protected Date parseDateJdk11(String pubDate) throws ParseException {
```

Comment:
What is different about parsing date with jdk11? A javadoc would be kind here (especially as geonetwork date parsing/handling decisions are split across several classes like ISODate).

Reply:
Given that this approach probably works for java17 also; it may be better to call this parseDate; and rename the older case as parseDateJdk8?